scripts/generators/spirv_validation_generator.py

```diff
@@ -89,7 +89,10 @@ def createMapValue(self, name: str, enable: SpirvEnables, isExtension: bool) ->
         if enable.version is not None:
             # Version should be VK_VERSION_x_x as defined in header but need to get as VK_API_VERSION_x_x
             version = enable.version.replace('VK_VERSION', 'VK_API_VERSION')
-            out.append(f'{{{version}, nullptr, nullptr, ""}}')
+            if name == 'SPV_KHR_non_semantic_info':
+                out.append('{VK_API_VERSION_1_0, nullptr, nullptr, ""}')            
```

Comment:
nit: but simpler python for future
```suggestion
if name == 'SPV_KHR_non_semantic_info':
    version = 'VK_API_VERSION_1_0'
out.append(f'{{{version}, nullptr, nullptr, ""}}')
```